docs/en/04_Changelogs/4.11.0.md

```diff
@@ -19,64 +20,58 @@ title: 4.11.0 (unreleased)
 - [Bugfixes](#bugfixes)
 - [Dependency and internal API changes](#dependency-internal-api-changes)
 
-## Regression test and Security audit{#audit}
+## Regression test and Security audit {#audit}
```

Comment:
Further up, change `Users will recieve an email` to `Users will receive an email`

Reply:
done